sdk/python/kfp/dsl/for_loop_test.py

```diff
@@ -151,6 +151,12 @@ def test_loop_artifact_argument_from_single_pipeline_channel_raises_error(
         self.assertRaises(ValueError,
                           for_loop.LoopArtifactArgument.from_pipeline_channel,
                           channel)
+        with self.assertRaisesRegex(
+                ValueError,
+                'Cannot iterate over a single Artifact using `dsl.ParallelFor`. Expected a list of Artifacts as argument to `items`.'
```

Comment:
Since `assertRaisesRegex` expects a regex rather than a literal string, we'll need use a raw string literal and escape all special regex characters:

```python
r'Cannot iterate over a single Artifact using `dsl\.ParallelFor`\. Expected a list of Artifacts as argument to `items`\.'
```

Reply:
Done